<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrugsDB::AtcPage</class>
 <widget class="QWidget" name="DrugsDB::AtcPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QTreeView" name="atcView"/>
   </item>
   <item row="0" column="0">
    <widget class="Utils::QButtonLineEdit" name="search"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::QButtonLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/widgets/qbuttonlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
